### PR TITLE
Installation docs. First pass review and changes. 

### PR DESCRIPTION
**Pull request by jhkrug**
A few points, marked with TODO: need a closer look.

### CODE REVIEW

File: docs/installation/install_epinio.md [hunk 1]
```diff
@@ -74,50 +79,59 @@ helm upgrade --install cert-manager jetstack/cert-manager --namespace cert-manag
...
-<details><summary>With "Let's Encrypt" certificates</summary>
-<p>To generate trusted TLS certificates with "Let's Encrypt" for your public domain set
-`.Values.global.tlsIssuer` to `letsencrypt-production` and your e-mail as value for
-`.Values.global.tlsIssuerEmail` key.</p>
+
+<details>
+
+<summary>Installation with "Let's Encrypt" certificates</summary>
+
+To generate trusted TLS certificates with "Let's Encrypt" for your public domain set `.Values.global.tlsIssuer` to `letsencrypt-production` and your e-mail to the value for the `.Values.global.tlsIssuerEmail` key.
```
Comment (andreas-kupries) [inline docs/installation/install_epinio.md:122]:
```suggestion
To generate trusted TLS certificates with "Let's Encrypt" for your public domain set `.Values.global.tlsIssuer` to `letsencrypt-production` and the value for the `.Values.global.tlsIssuerEmail` key to your e-mail.
```

File: docs/installation/install_epinio.md [hunk 2]
```diff
@@ -126,75 +140,85 @@ Ingress controller.
...
+- [Install on Public Cloud](other_inst_scenarios/install_epinio_on_public_cloud.md) — Install Epinio on Public Cloud cluster
+- [Install on RKE2](other_inst_scenarios/install_epinio_on_rke.md) — Install Epinio on Rancher RKE2 cluster
+- [Install on K3d](other_inst_scenarios/install_epinio_on_k3d.md) — Install Epinio on K3d cluster
+- [Install on K3s](other_inst_scenarios/install_epinio_on_k3s.md) — Install Epinio on K3s cluster
+- [Install on Rancher Desktop](other_inst_scenarios/install_epinio_on_rancher_desktop.md) — Install Epinio on Rancher Desktop
+- [Install on EKS](other_inst_scenarios/install_epinio_on_eks.md) — Install Epinio on AWS EKS cluster
 
-> *NOTE*: The Public Cloud How-to lists the three major Cloud providers but Epinio can run on any Kubernetes cluster.
+:::note
+The Public Cloud How-to describes the three major cloud providers but Epinio can run on any Kubernetes cluster.
```
Comment (andreas-kupries) [inline docs/installation/install_epinio.md:181]: Maybe link to this how to ?

File: docs/installation/install_epinio.md [hunk 3]
```diff
@@ -126,75 +140,85 @@ Ingress controller.
...
+:::
 
 ## Internal Epinio components
+
 ### Kubed
 
 Kubed is installed as a subchart when `.Values.kubed.enabled` is `true` (default).
-If you already have kubed running, you can skip the installation by setting
+If you already have kubed, you can skip installation by setting
```
Comment (andreas-kupries) [inline docs/installation/install_epinio.md:189]:
```suggestion
If you already have `kubed`, you can skip installation by setting
```

File: docs/installation/install_epinio.md [hunk 4]
```diff
@@ -126,75 +140,85 @@ Ingress controller.
...
 
-In addition to Minio, Epinio offers [s3gw](https://s3gw.io/) as another internal S3 compatible storage. It is installed when `.Values.minio.enabled` is set to `false` and `.Values.s3gw.enabled` is set to `true`.
+In addition to Minio, Epinio offers [s3gw](https://s3gw.io/) as another S3 compatible store.
+It is installed when `.Values.minio.enabled` is set to `false` and `.Values.s3gw.enabled` is set to `true`.
 
-Both choices for internal S3 compatible storage can be configured to use a user-defined storageClass. If no StorageClass is defined, the default storageClass is used. For Minio the custom storageClass is set via the value of `.Values.persistance.storageClass`. For s3gw the custom storageClass is set via the value of `.Values.s3gw.storageClass.name`.
+Both choices for internal S3 compatible storage can be configured to use a user-defined storageClass.
+If no StorageClass is defined, the default storageClass is used.
+Using Minio set the custom storageClass by the value of `.Values.persistance.storageClass`.
+Using s3gw set the custom storageClass by the value of `.Values.s3gw.storageClass.name`.
```
Comment (andreas-kupries) [inline docs/installation/install_epinio.md:204]:
```suggestion
When using s3gw set the custom storageClass as the value of `.Values.s3gw.storageClass.name`.
```

(and minio equivalently)

File: docs/installation/install_epinio.md [hunk 5]
```diff
@@ -126,75 +140,85 @@ Ingress controller.
...
 
-Any external S3 compatible solution can be used instead by setting `.Values.minio.enabled` value to `false` (`.Values.s3gw.enabled` is `false` by default) and using [the values under `s3`](https://github.com/epinio/helm-charts/blob/b389a4875af9f03b484a911c49a14f834ba04b64/chart/epinio/values.yaml#L44) to point to the desired S3 server.
+<!--TODO: Not sure about using a hashed blob, is that OK? Presumably to guarantee a line number reference?-->
```
Comment (andreas-kupries) [inline docs/installation/install_epinio.md:206]: Yes. A non-hashed reference, i.e. by branch name can shift what it points to, as the file changes on the branch.

File: docs/installation/install_epinio.md [hunk 6]
```diff
@@ -126,75 +140,85 @@ Ingress controller.
...
 [Dex](https://dexidp.io) OpenID Connect Provider is installed as a subchart when `.Values.global.dex.enabled` is set to `true` (default).
 
-In case you don't need to use an identity provider you may set the value to `false` and use only local epinio users. [OIDC Authentication](../references/authentication_oidc.md) provides more information.
+If you don't need to use an identity provider, set the value to `false` and use only local Epinio users. [OIDC Authentication](../references/authentication_oidc.md) has more information.
 
 ### Container Registry
 
 When Epinio builds a container image for an application from source, it needs
-to store that image to a container registry. Epinio installs a container registry
+to store that image in container registry. Epinio installs a container registry
```
Comment (andreas-kupries) [inline docs/installation/install_epinio.md:218]:
```suggestion
to store that image in a container registry. Epinio installs a container registry
```

**Review by andreas-kupries — COMMENTED** [review]
Some suggestions, answers, ...

File: docs/installation/install_epinio.md (hunk 2 above)
Comment (jhkrug) [reply docs/installation/install_epinio.md:181]: I will do.

**Review by andreas-kupries — APPROVED** [review]

File: docs/installation/install_epinio.md [hunk 7]
```diff
@@ -96,103 +108,114 @@ kubectl patch storageclass local-path -p '{"metadata": {"annotations":{"storagec
...
-should be a wildcard `*.` enabled domain, pointing to the IP address of your running
-Ingress controller.
+The only mandatory field is the `.Values.global.domain` which should have the value of a wildcard `*.` enabled domain.
+It should point to the IP address of your running Ingress controller.
+
+:::tip
+
+- To use a non-default IngressClass you need to specify it using `--set ingress.ingressClassName=<className>`
+
+- If you receive `Entity Too Large` errors when [uploading](https://github.com/kubernetes/ingress-nginx/blob/main/docs/user-guide/nginx-configuration/annotations.md#custom-max-body-size) application source code into Epinio, you need to increase `proxy-body-size` with `--set 'ingress.annotations.nginx\.ingress\.kubernetes\.io/proxy-body-size=1000m'`
```
Comment (andreas-kupries) [inline docs/installation/install_epinio.md:138]:
```suggestion
- If you receive `Entity Too Large` errors when [uploading](https://github.com/kubernetes/ingress-nginx/blob/main/docs/user-guide/nginx-configuration/annotations.md#custom-max-body-size) application source code into Epinio, you need to increase the `proxy-body-size` with `--set 'ingress.annotations.nginx\.ingress\.kubernetes\.io/proxy-body-size=1000m'`
```

nit.

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
Small nit, otherwise looks ok to me.

**Review by andreas-kupries — APPROVED** [review]

**Pull request merged**